PowerEditor/src/Parameters.cpp

```diff
@@ -1012,6 +1013,16 @@ NppParameters::NppParameters()
 	//Load initial accelerator key definitions
 	initMenuKeys();
 	initScintillaKeys();
+
+	// Rescale default panels dimensions according to the current DPI
+	_nppGUI._dockingData._leftWidth = _dpiManager.scaleX(_nppGUI._dockingData._leftWidth);

```

Comment:
@ozone10 Please correct me if I say somethings wrong.

@xomx 
`DPIManager` is deprecated and it will be removed in the future. Please use `DPIManagerV2` instead.

And the value of docking data should not be modified here, but when it is used IMO.

For example, instead of adding the above line, please rather use the following line:
`_dockingManager.setDockedContSize(CONT_LEFT, DPIManagerV2::scale(nppGUI._dockingData._leftWidth, dpi));`
instead of https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/Notepad_plus.cpp#L798

Reply:
@donho @xomx 
when using `DPIManagerV2` panels sizes need to be saved unscaled. And when loaded they need to be scaled according to dpi on screen they currently are. And because of this multi-screen situation, especially for floating panels, some code from `Parameters.*` needs to be moved to "docking" files.

But since this will require a lot of work, simple "in place" replacing old `DPIManager` with `DPIManagerV2` with correct dpi should be enough for current dpi manifest.

Reply:
@donho @ozone10 
Thanks for your hints.

To not further complicate this PR, I will leave the mentioned right Docking Manager data handling to @ozone10 (storing only unscaled data, otherwise it will slowly grow with every N++ relaunch without a user-initiated dimension change, right?). Inherent data needed for this PR are not stored to the config.xml via Parameters.cpp handling anyway, the base for the minimum panel sizes calculations is the DockingCont.h constant HIGH_CAPTION.

So now this PR uses the requested DPIManagerV2 and the initial scale adjustment was moved from the NppParameters ctor to the suggested Notepad_plus::init.

At runtime, watching for the possible scaling changes has been already added by me to the DockingManager::runProc -> case DMM_MOVE_SPLITTER.